Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -295,6 +295,22 @@ open class Kommunicate: NSObject, Localizable {
         navVC.modalPresentationStyle = .fullScreen
         viewController.present(navVC, animated: true, completion: nil)
     }
+    
+    /**
+      Launch chat list from a ViewController.
+      - Parameters:
+      - viewController: ViewController from which the chat list  will be added as child vc
+      - rootView: view container where chat will be loaded.
+      */
+     @objc open class func embedConversationList(from viewController: UIViewController, on rootView: UIView) {
+         let conversationVC = conversationListViewController()
+         let navVC = KMBaseNavigationViewController(rootViewController: conversationVC)
+         navVC.willMove(toParent: viewController)
+         navVC.view.frame = rootView.bounds
+         rootView.addSubview(navVC.view)
+         viewController.addChild(navVC)
+         navVC.didMove(toParent: viewController)
```

Comment:
We have similar code in the new `openChatOnView` function, try to reuse it.

Reply:
done